src/components/NoMatch.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const NoMatch = () => {
+  return (
+    <div>
+      <h1>404 Not Found</h1>
```

Comment:
move it to the center, and change
![image](https://user-images.githubusercontent.com/38506213/112024965-fdf23000-8b34-11eb-96df-c72925da6fd3.png)